ui/admin/tests/acceptance/policy/list-test.js

```diff
@@ -185,12 +181,16 @@ module('Acceptance | policies | list', function (hooks) {
     });
     urls.policy = `${urls.policies}/${instances.policy.id}`;
 
-    await click(`[href="${urls.policies}"]`);
-    await click(DROPDOWN_BUTTON_SELECTOR);
+    await click(commonSelectors.HREF(urls.policies));
+    await click(commonSelectors.TABLE_FIRST_ROW_ACTION_DROPDOWN);
+
+    assert
+      .dom(commonSelectors.TABLE_FIRST_ROW_ACTION_DROPDOWN_ITEM_LINK)
+      .exists()
+      .hasText('Edit');
+
+    await click(commonSelectors.TABLE_FIRST_ROW_ACTION_DROPDOWN_ITEM_LINK);
```

Comment:
❤️ This reads so much better now. I can actually tell what we are acting on and asserting against.